### DISCUSSION TITLE
Issue annotating EC output

### DISCUSSION THREAD

**Discussion by g-raymo** [cafferychen777/ggpicrust2#95] [Q&A]
I am attempting the annotation step in the pipeline outlined below.
metacyc_daa_annotated_results_df <- pathway_annotation(pathway = "MetaCyc", daa_results_df = metacyc_daa_results_df, ko_to_kegg = FALSE)

When I execute this command in my code containing my own vars, I receive the following error message:
Error in `$<-.data.frame`(`*tmp*`, "description", value = character(0)) : 
  replacement has 0 rows, data has 1

Please advise how best to rectify.

Thanks!

**Comment by cafferychen777** [comment]
Dear @g-raymo,

Thank you for reaching out with your question. The error you're encountering typically occurs when there is a mismatch in the number of rows between the data frame being modified and the replacement value.

To better assist you, could you please provide the complete script that you're using, as well as the data files involved in this step? This will help me understand the context of the error and provide a more accurate solution.

You can send the script and data files to my email at cafferychen7850@gmail.com. Once I have a chance to review them, I'll be able to offer more specific guidance on how to resolve the issue.

Best regards,
Caffery Yang
